airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -350,8 +349,8 @@ private SyncCheckConnectionFailure checkConnections(final GenerateInputActivity.
       log.info("SOURCE CHECK: Skipped");
     } else {
       log.info("SOURCE CHECK: Starting");
-      final StandardCheckConnectionOutput sourceCheckResponse = runMandatoryActivityWithOutput(checkActivity::run, checkSourceInput);
-      if (sourceCheckResponse.getStatus() == Status.FAILED) {
+      final ConnectorJobOutput sourceCheckResponse = runMandatoryActivityWithOutput(checkActivity::run, checkSourceInput);
```

Comment:
This activity now has a different output type, being called from the ConnectionManagerWorkflow. I know when it comes to changing the inputs we have to do some version checks - is there anything special to do here because of this output change?

Reply:
@benmoriceau is the expert here, but since these are short-lived activities (vs syncs which can take days and span multiple deployments), maybe we don't need to do a version check?

Reply:
unfortunatelly we need a new version here. You can check that by updating the workflowHistory in `WorkflowReplayingTest` by something generated from master. Here the test doesn't fail because it is already controlled by a version. The time that an activity takes to run doesn't influence how likely it can have a versioning issue. The main factor is where in the workflow the activity is run. For example the last activity is less likely to have a version issue because the workflow will terminate/continue as new after it while for the first activity it is more likely that the workflow will be unloaded from memory and potentially replayed.

Reply:
@benmoriceau Ah! Thanks for pointing out the `WorkflowReplayingTest` - it did indeed fail and was super helpful for getting this working. I've updated this to consider a version in [6305fe4](https://github.com/airbytehq/airbyte/pull/14715/commits/6305fe4958a2999108e56cb5c531b4a389fe638c)

Reply:
Thanks @pedroslopez, about that. We have one last issue to ensure that the workflow are not block because of a version. After that we will clear all the version. For the future we will still keep the version but it will be removal in a timely manner.